cl/audio/models.py

```diff
@@ -236,7 +243,7 @@ def as_search_dict(self):
                 "duration": self.duration,
                 "source": self.source,
                 "download_url": self.download_url,
-                "local_path": str(getattr(self, "local_path_mp3", None)),
+                "local_path": deepgetattr(self, "local_path_mp3.name", None),
```

Comment:
This `deepgetattr` is cute - and something I would use but its totally a black box